.githooks/pre-commit

```diff
@@ -2,28 +2,53 @@
 
 echo "Running FarmData2 pre-commit checks..."
 
-# Prevent commits to the production branch
+# Prevent commits to the production and development branches
 BRANCH="$(git rev-parse --abbrev-ref HEAD)"
-if [ "$BRANCH" = "production" ]; then
+if [ "$BRANCH" = "production" ] || [ "$BRANCH" = "development" ]; then
   echo
-  echo "You cannot commit directly to the production branch."
-  echo "The production branch should only be updated by merging development into it."
-  echo "This operation should only be performed by maintainers."
-  echo
-  exit 255
-fi
-
-if [ "$BRANCH" = "development" ]; then
-  echo
-  echo "You cannot commit directly to the development branch."
-  echo "Instead create a feature branch and commit your changes to it."
+  echo "You cannot commit directly to the $BRANCH branch."
+  echo "Instead, create a feature branch and commit your changes to it."
   echo "Then push your feature branch to GitHub and create a pull request."
   echo
   exit 255
 fi
 
-# For all other branches, run the pre-commit checks using lint-staged.
-if npm run check:staged; then
+# Initialize an array to keep track of tests that do not need to be run again.
+declare -a skipTests=()
+
+# Check for .only in .cy.js files and mark .vue file tests
```

Comment:
See my comment in the overall review comment about possibly using an eslint rule for this instead.